.gitignore

```diff
@@ -24,6 +25,8 @@ settings.py
 static/javascripts/metadata_tooltips.js
 ui_library
 ~$
+settings.py.local
```

Comment:
Is this similar to the tests.py file?

Reply:
I am ignoring the backup file for settings.py since I do not need to track matomo most of the time and I needed to change my settings.py.  I can remove this if you want and copy it somewhere else.  I do not want to commit it.

Reply:
I've removed this from ignoring, though I don't believe it hurts anything to ignore.